Got ERESOLVE on `npm i`, fixed by upgrading @vitejs/vue-plugin 5 -> 6
Also updated npmDepsHash (needed whenever package-lock.json changes).

It builds and runs now but I haven't tested the UI.

--- a/.husky/pre-commit
+++ b/.husky/pre-commit
@@ -1,1 +1,1 @@
-npm run pre-commit
+pnpm run pre-commit

--- a/README.md
+++ b/README.md
@@ -214,8 +214,8 @@ MIX_ENV=prod mix compile
 
 ### Install and Build Frontend Assets:
 ```bash
-npm install
-npm run build
+pnpm install
+pnpm run build
 MIX_ENV=prod mix phx.digest
 ```
 

